CHANGELOG.md

```diff
@@ -11,12 +11,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changes
 
+- BREAKING: move `to_primitive` to a more fittingly named `mex.backend.serialization`
+- BREAKING: swap `INDEXABLE_MODEL_CLASSES_BY_NAME` for `ALL_MODEL_CLASSES_BY_NAME`
+  to also include non-indexable models (namely: merged models)
+- BREAKING: rename `fetch_extracted_data` to a more consistent `fetch_extracted_data`
```

Comment:
these are the same names. I think you wanted `fetch_extracted_items` here.

Reply:
ups, yeah ad63e9b